src/test-metrics.jl

```diff
@@ -111,14 +111,19 @@ Combines the merics results from a `sub` into its `parent`
 """
 function combine_test_metrics(parent, sub)
     # nothing to do
+   # we should always return the new parent, in case we've replaced the parent.
+   # Note: we can replace the parent only if it's a subtype of `TestMetrics`
```

Comment:
```suggestion
    # we should always return the new parent, in case we've replaced the parent.
    # Note: we can replace the parent only if it's a subtype of `TestMetrics`
```